src/infra/service-infrastructure.md

```diff
@@ -25,16 +25,17 @@ The [Rust Log Analyzer](https://github.com/rust-lang/rust-log-analyzer)
 analyzes CI build logs to extract error messages and posts them to the pull
 request. It is run by [TimNN](https://github.com/TimNN).
 
-## Homu
+## Homu / bors
 
-[Homu](https://github.com/rust-lang/homu/) is a bot
-([bot user account](https://github.com/bors)) which manages pull requests.
+[Homu](https://github.com/rust-lang/homu/) is a bot often referred to as "bors" due
+to the name of its [bot user account](https://github.com/bors) which manages pull requests.
 Approved pull requests are placed in
 [a queue](http://buildbot2.rust-lang.org/homu/queue/rust) from which tests are
 run.
 
 Documentation on homu commands can be found
 [here](http://buildbot2.rust-lang.org/homu/).
+The homu source code and issue tracker lives [here](https://github.com/rust-lang/homu).
```

Comment:
Is that really the issue tracker to point people to? I do not know.

Reply:
Oh never mind, I only now noticed the first word of the paragraph links to this already.